config/webpack/config.production.js

```diff
@@ -30,7 +30,7 @@ module.exports = env => ({
     new HtmlPlugin({
       template: './assets/index.html'
     }),
-    new CopyPlugin([{ from: `config/env/env.*.json` }]),
+    new CopyPlugin([{ from: `config/env/env.*.json`, to: 'env.json' }]),
```

Comment:
Got it but done on purpose : should copy all the `env.*.json` files then manually (or better automatically with deployment tool) move the proper one according to the targeted environment (local, prod, ...).
Anyway, this change would not work if we've got `env.local.json` and `env.prod.json` files : the plugin copies the first one and silently ignores the second (good) one.

In my opinion, 2 ways to handle this :
1. Add `--env.ENV=prod`  option to _prod_ script (in `package.json`)
Then inject the env value to target the right file in webpack prod config :
```javascript
new CopyPlugin([
      { from: `config/env/env.${env.ENV}.json`, to: 'env.json' }
    ])
```
2. Or simply copy a `env.prod.json` :
```javascript
new CopyPlugin([
      { from: `config/env/env.prod.json`, to: 'env.json' }
    ])
```

Reply:
I prefer 2 :) more simple :))